conda/dev-environment-win.yml

```diff
@@ -0,0 +1,31 @@
+name: csp-dev
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+ - compilers
```

Comment:
this is unsufficient and unsorted.

Reply:
Oh I see. I will see what else to install on Windows.